vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/validation/rules/VGSInfoValidationRule.kt

```diff
@@ -0,0 +1,70 @@
+package com.verygoodsecurity.vgscollect.view.card.validation.rules
+
+class VGSInfoValidationRule(
```

Comment:
@Endorf Maybe if we have Builder we should make the constructor private? Because now dev can create `VGSInfoValidationRule("some_regex", arrayOf(-1, 100, -3))` etc.